lib/cli.rb

```diff
@@ -26,36 +26,63 @@ def format_board_for_display(board_state)
     board_str = "\n"
 
     board_state.each_with_index do |row, row_idx|
-      board_str += generate_board_row(row, row_idx)
+      board_str += generate_board_row(row, row_idx, board_state.length)
     end
 
     board_str + "\n"
   end
 
-  def generate_board_row(row, row_idx)
+  def generate_board_row(row, row_idx, width)
     row_str = ''
 
     row.each_with_index do |square, square_idx|
-      square_text = get_square_text(square, row_idx, square_idx)
+      square_text = get_square_text(square, row_idx, square_idx, width)
 
       row_str += " #{square_text} "
-      row_str += square_idx < 2 ? '|' : "\n"
+      row_str += square_idx < (width - 1) ? '|' : "\n"
     end
 
     # add row divider
-    row_str += "-----------\n" if row_idx < 2
+    row_str += divider(width: width) if row_idx < (width - 1)
 
     row_str
   end
 
-  def get_square_text(square, row_idx, square_idx)
+  def get_square_text(square, row_idx, square_idx, width)
+    text = square
+
     if square.nil?
-      row_base = row_idx * 3
+      row_base = row_idx * width
 
       # Add one because it's not zero-based numbering for readability
-      return row_base + square_idx + 1
+      text = (row_base + square_idx + 1).to_s
+    end
+
+    text = pad_str(text: text, width: width)
+
+    text
+  end
+
+  def pad_str(text:, width:)
```

Comment:
Any better name ideas for this one? thnx :)

Reply:
does `center_square_text` sound better?